### PR TITLE
Category Browser

### PR DESCRIPTION
**Pull request by Kwantux**
Adds a category browser to the waystone menu

### CODE REVIEW

**Comment by Kwantux** [comment]
TODO:
- [x] Bedrock UI
- [x] Add setup menu as it was in the old config menu

File: waystones-paper/src/main/java/com/kalimero2/team/waystones/paper/ui/JavaScreens.java [hunk 1]
```diff
@@ -25,10 +26,7 @@
 import org.jetbrains.annotations.NotNull;
 import org.jetbrains.annotations.Nullable;
 
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.List;
-import java.util.UUID;
```
Comment (byquanton) [inline waystones-paper/src/main/java/com/kalimero2/team/waystones/paper/ui/JavaScreens.java:31]: Pfui, fixe mal deine IDE

File: waystones-paper/src/main/java/com/kalimero2/team/waystones/paper/ui/WaystonesScreen.java [hunk 2]
```diff
@@ -52,6 +53,22 @@ public void menu(Player player, @Nullable StoredWaystone waystone) {
         }
     }
 
+    public void browse(Player player, Category category) {
+        if (plugin.isBedrockPlayer(player)) {
+            //TODO: Implement Browse feature for Bedrock
```
Comment (byquanton) [inline waystones-paper/src/main/java/com/kalimero2/team/waystones/paper/ui/WaystonesScreen.java]: Okay?

File: waystones-paper/src/main/java/com/kalimero2/team/waystones/paper/ui/WaystonesScreen.java [hunk 3]
```diff
@@ -52,6 +53,22 @@ public void menu(Player player, @Nullable StoredWaystone waystone) {
...
+        } else {
+            java.browse(player, category);
+        }
+    }
+
+    public void browseCategorySelection(Player player) {
+        if (plugin.isBedrockPlayer(player)) {
+            //TODO: Implement Browse feature for Bedrock
```
Comment (byquanton) [inline waystones-paper/src/main/java/com/kalimero2/team/waystones/paper/ui/WaystonesScreen.java]: Okay?

File: waystones-paper/src/main/java/com/kalimero2/team/waystones/paper/ui/WaystonesScreen.java (hunk 2 above)
Comment (byquanton) [reply waystones-paper/src/main/java/com/kalimero2/team/waystones/paper/ui/WaystonesScreen.java]: Normales Menü öffnen

File: waystones-paper/src/main/java/com/kalimero2/team/waystones/paper/ui/WaystonesScreen.java (hunk 3 above)
Comment (byquanton) [reply waystones-paper/src/main/java/com/kalimero2/team/waystones/paper/ui/WaystonesScreen.java]: Normales Menü öffnen

**Pull request merged**